trunk/SUAVE/Methods/Flight_Dynamics/Dynamic_Stability/single_point_aero_derivatives.py

```diff
@@ -0,0 +1,188 @@
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+# single_point_aero_derivatives.py
+# 
+# Created:   Aug 2021, R. Erhard
+# Modified: 
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Data
+from SUAVE.Plots.Mission_Plots import *  
+
+import numpy as np 
+from copy import deepcopy
+
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+def single_point_aero_derivatives(analyses,vehicle, state, h=0.01): 
+    """This function takes an aircraft with analyses setup and initial state,
+    and computes the aerodynamic derivatives about the initial state.
+    
+    Assumptions:
+       Linerarized equations are used for each state variable
+
+    Source:
+      N/A
+
+    Inputs:
+      vehicle                SUAVE Vehicle 
+      state                  Initial state of vehicle (after running mission segment of interest)
+
+    Outputs: 
+       aero_derivatives      Aerodynamic derivatives wrt perturbed state variables
+
+    Properties Used:
+       N/A
+     """
+    
+    # extract states from prior mission segment
+    alpha           = state.conditions.aerodynamics.angle_of_attack
+    throttle        = state.conditions.propulsion.throttle
+    velocity        = state.conditions.freestream.velocity
+    velocity_vector = state.conditions.frames.inertial.velocity_vector
+    
+    # ----------------------------------------------------------------------------
+    # Perturb each state variable
+    # ----------------------------------------------------------------------------
+    # Alpha perturbation
+    perturbed_state = deepcopy(state)
+    alpha_plus      = alpha + 0.5*h
+    alpha_minus     = alpha - 0.5*h
+    
+    perturbed_state.conditions.aerodynamics.angle_of_attack = alpha_plus
+    results_Alpha_plus = evaluate_aero_results(analyses,vehicle,perturbed_state)
```

Comment:
I know aaron is working on this same thing. He's batching it out. You should be able to get all derivatives in one solve: [initial, initial + alpha, initial -alpha, initial +throttle...]

Let's talk if you have questions. Maybe we can chat with aaron too.